content/en/flux/cheatsheets/oci-artifacts.md

```diff
@@ -396,6 +396,87 @@ Note that keyless verification is an **experimental feature**, using
 custom root CAs or self-hosted Rekor instances are not currently supported.
 {{% /alert %}}
 
+### Notary Workflow example
+
+Add a certificate to the trusted certificates store:
+
+```shell
+notation cert add --type ca --store <store> <cert>
+```
+
+It is also possible to generate a test certificate:
+
+```shell
+# Generate a certificate and RSA key pair
+notation cert generate-test valid-example
+```
+
+Push and sign the artifact using the certificate's private key:
+
+```shell
+flux push artifact oci://ghcr.io/stefanprodan/manifests/podinfo:$(git tag --points-at HEAD) \
+	--path="./kustomize" \
+	--source="$(git config --get remote.origin.url)" \
+	--revision="$(git tag --points-at HEAD)/$(git rev-parse HEAD)"
+
+notation sign ghcr.io/stefanprodan/manifests/podinfo:$(git tag --points-at HEAD) -k <key>
+```
+
+Create a `trustpolicy.json` file:
+
+```json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "<policy-name>",
+            "registryScopes": [ 
+                "ghcr.io/stefanprodan/manifests/podinfo"
+             ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:<store-name>" ],
+            "trustedIdentities": [
+                "*"
```

Comment:
If you're `strict` it's better to specify the `trustedIdentities` rather than using a wildcard. Using `stefanprodan`'s podinfo it would be better to use the following:

```suggestion
                "x509.subject: C=RO, ST=BU, L=Bucharest, O=Notary, CN=stefanprodan.com"
```